README.md

```diff
@@ -1 +1,10 @@
 # kintro
+
+[![GitHub license](https://img.shields.io/github/license/Naereen/StrapDown.js.svg)](https://github.com/neckbeard-io/kintro/blob/master/LICENSE)
+[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/neckbeard-io/kintro/main.svg)](https://results.pre-commit.ci/latest/github/neckbeard-io/kintro/main)
+[![CI](https://github.com/neckbeard-io/kintro/actions/workflows/check.yml/badge.svg)](https://github.com/neckbeard-io/kintro/actions/workflows/check.yml)
+[![GitHub latest commit](https://badgen.net/github/last-commit/Naereen/Strapdown.js)](https://GitHub.com/neckbeard-io/kintro/commit/)
```

Comment:
This one looks wrong? Unless I'm missing how it's coming up with that date. github.com should probably be case smashed too due to OCD.

Reply:
Changed it up